formSchemas.js

```diff
@@ -0,0 +1,13 @@
+const loginSchema = {
+    code: {
+      in: ['body'],
+      isLength: {
+        errorMessage: 'Must be 8 characters',
+        options: { min: 8 }
+      },
+    },
+  }
+
+  module.exports = {
+    loginSchema
+  }
```

Comment:
Small point, but I'm going to figure out how to get trailing newlines to be automatically inserted as they were on past projects.